lib/order.rb

```diff
@@ -0,0 +1,68 @@
+require_relative 'menu'
+require_relative 'confirmation'
+require 'dotenv'
+Dotenv.load
+
+class Order
+
+  attr_reader :selection, :menu, :prices, :total, :orders, :confirmation
+
+  def initialize(menu = Menu.new)
+  @menu = menu
+  @orders = []
+  @selection = {}
+  @prices = []
+  end
+
+  def start
+    puts @menu.introduction
+    puts @menu.show_menu
+  end
+
+  def select_order(item, amount)
+    @selection = { item.to_sym => amount}
+    add_to_orders
+    return "#{amount} x #{item} added to your order!"
+  end
+
+  def get_price
+    @orders.each do |order|
+    order.each do |item, amount|
+    @price = menu.dishes[item]
+    puts "#{amount} x #{item.to_sym} at £#{@price} each = £#{@price *= amount}"
+    @prices << @price
```

Comment:
Yes, get_price seems like it could be a good method to change so it doesn't have any side effects (that is, it doesn't rely on changing `@prices`).  Is `get_price` required? Would just having `total_price` be enough? Then total_price could just calculate and return the price and there would be no need for `@prices`.